cypress/support/step_definition/news/readNewsSteps.spec.js

```diff
@@ -0,0 +1,27 @@
+import News from '../../pages/news/pageNews'
+
+const news = new News()
+
+//Scenario: Reading a school news on the dashboard
```

Comment:
Remove unnecessary comments

Reply:
so the scenario is also added in the step definition as in the feature file of it.